pkg/api/routes.go

```diff
@@ -193,7 +193,7 @@ func (rh *RouteHandler) SetupRoutes() {
 	// Preconditions for enabling the actual extension routes are part of extensions themselves
 	ext.SetupSearchRoutes(rh.c.Config, prefixedRouter, rh.c.StoreController, rh.c.MetaDB, rh.c.CveInfo,
 		rh.c.Log)
-	ext.SetupImageTrustRoutes(rh.c.Config, prefixedRouter, rh.c.Log)
+	ext.SetupImageTrustRoutes(rh.c.Config, rh.c.MetaDB, prefixedRouter, rh.c.Log)
```

Comment:
Nitpicking, maybe the router should be the 2nd parameter and the db the 3rd, similar to the other methods.